zerver/webhooks/linear/tests.py

```diff
@@ -0,0 +1,85 @@
+from zerver.lib.test_classes import WebhookTestCase
+
+
+class HookTests(WebhookTestCase):
```

Comment:
@timabbott is this how we're naming webhook tests now? Its been a while since I did one, but thought we still put the integration name in there.

Reply:
Yes you are right I missed the integration name over here, will fix that soon.